test/connectors/ERSConnectorSpec.scala

```diff
@@ -49,47 +53,42 @@ class ERSConnectorSpec extends UnitSpec with MockitoSugar with OneAppPerSuite wi
 
   lazy val schemeInfo = SchemeInfo("XA1100000000000", DateTime.now, "1", "2016", "EMI", "EMI")
 
-  "calling sendData" should {
+  lazy val mockHttp = mock[HttpPost]
+  lazy val mockMetrics: Metrics = mock[Metrics]
 
-    lazy val mockHttp = mock[HttpPost]
-    lazy val mockMetrics: Metrics = mock[Metrics]
+  lazy val ersConnector: ErsConnector = new ErsConnector {
+    override lazy val metrics: Metrics = mockMetrics
 
-    lazy val schemeInfo1 = SchemeInfo("XA1100000000000", DateTime.now, "1", "2016", "QQQ", "QQQ")
+    override def httpPost: HttpPost = mockHttp
 
-    lazy val callbackData = CallbackData(
-      collection = "collection",
-      id = "someid",
-      length = 1000L,
-      name = Some(Fixtures.firstName),
-      contentType = Some("content-type"),
-      sessionId = Some("testId"),
-      customMetadata = Some(Json.obj("sessionId" -> "testId")), noOfRows = None)
+    override def httpGet: HttpGet = mock[HttpGet]
 
-    lazy val ersConnectorUnderTest: ErsConnector = new ErsConnector {
+    override def ersUrl = "ers-returns"
 
-      override lazy val metrics: Metrics = mockMetrics
+    override def validatorUrl = "ers-file-validator"
+  }
 
-      override def httpPost: HttpPost = mockHttp
+  lazy val data: JsObject = Json.obj(
+    "schemeRef" -> "XA1100000000000",
+    "confTime" -> "2016-08-05T11:14:43"
+  )
 
-      override def httpGet: HttpGet = mock[HttpGet]
+  "validateFileData" should {
```

Comment:
missing the content of three tests

Reply:
*facepalm* I am writing some wiremock tests now. Dunno how I missed this, sorry.

Reply:
Not to worry :)